docs/conf.py

```diff
@@ -258,7 +258,7 @@ def __getattr__(cls, name):
 tmt.plugins._explore_export_directory()
 
 # Generate stories
-tree = tmt.Tree(logger=tmt.Logger.create(), path='.')
+tree = tmt.Tree(logger=tmt.Logger.create(), path=Path.cwd())
 
```

Comment:
not sure if make sense like this, but Tree expects Path not str.

Reply:
Correct, this is a genuine bug, it should be `Path`.